### PR TITLE
Add barchart

### PR DESCRIPTION
**Pull request by fanny**
#### What is the purpose of this pull request?
as the title says.

#### How should this be manually tested?
`yarn install` && `yarn start`

#### Screenshots or example usage
image

#### Types of changes

- [ ] Bug fix (a non-breaking change which fixes an issue)
- [x] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

**Comment by fanny** [comment]
Yep, @Klynger, I agree with you, I talked with @kevinch about it. I thought of making something similar to [this](https://uber.github.io/react-vis/) like a "vtex-vis". But we decide to put here, and after more discussions with the style guide team, we will move to another structure, maybe. I feel that the kind of these components is different from the others, that the style guide already has, and makes sense it be an isolated module.
I will aprecciate have your support to begin a discussion about this in styleguide channel

File: react/components/EXPERIMENTAL_Charts/LineChart/helpers.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { defaultProps } from './constants'
+import { merge } from '../helpers'
+
+const getLineDefaultProps = (userProps: LineProps)  => {
+    const lineConfigs = merge(defaultProps, userProps)
+    return { lineConfigs }
+}
+
```
Comment (kevinch) [inline react/components/EXPERIMENTAL_Charts/LineChart/helpers.ts]: Remove extra line break.

File: react/components/EXPERIMENTAL_Charts/LineChart/index.tsx [hunk 2]
```diff
@@ -11,46 +11,49 @@ import {
...
   dataKeys,
   xAxisKey,
   schema,
-  formatter
+  formatter,
+  lineProps
 }) => {
-  const { configs } = getDefaultProps(schema); 
+  const { configs } = getChartDefaultProps(commonDefaultProps, schema)
+  console.log(configs)
```
Comment (kevinch) [inline react/components/EXPERIMENTAL_Charts/LineChart/index.tsx]: Remove console.log.

File: react/components/EXPERIMENTAL_Charts/LineChart/index.tsx [hunk 3]
```diff
@@ -11,46 +11,49 @@ import {
...
+  const { lineConfigs } = getLineDefaultProps(lineProps)
 
   return (
     <ResponsiveContainer {...configs.container}>
       <LineChartBase data={data}>
-        <XAxis dataKey={xAxisKey} {...configs.axis} />
-        <YAxis {...configs.axis}/>
+        <XAxis dataKey={xAxisKey} {...configs.xAxis}/>
```
Comment (kevinch) [inline react/components/EXPERIMENTAL_Charts/LineChart/index.tsx]: Try to use an automatic linter to keep consistency.
```suggestion
        <XAxis dataKey={xAxisKey} {...configs.xAxis} />
```

File: react/components/EXPERIMENTAL_Charts/typings.d.ts [hunk 4]
```diff
@@ -1,9 +1,29 @@
...
-  axis?: BaseAxisProps,
+type ChartSchema = {
+  xAxis?: BaseAxisProps,
+  yAxis?: BaseAxisProps, 
   container?: BaseContainerProps,
   grid?: BaseGridProps
-}
\ No newline at end of file
+}
+
```
Comment (kevinch) [inline react/components/EXPERIMENTAL_Charts/typings.d.ts]: Remove extra line break.

**Comment by fanny** [comment]
I had problems with the rebasing, so i move to #958

**Pull request closed without merging**